libr/util/time.c

```diff
@@ -197,3 +197,16 @@ R_API const char *r_time_to_string (ut64 ts) {
 	l = ts >> 20;
 	return r_time_stamp_to_str (l);
 }
+
+R_API char *r_asctime_r(const struct tm *tm, char *buf, size_t size) {
+	if (size < ASCTIME_BUF_MINLEN) {
+		size = ASCTIME_BUF_MINLEN;
+	}
```

Comment:
Line 202, 203, 204 should be deleted. I made a mistake while reading the documentation.